tools/data_faker/data_faker/statistics.py

```diff
@@ -177,8 +177,8 @@ def create_fake_data(self, cursor: DBCursor, args: argparse.Namespace) -> None:
                     category=BalanceType.ASSET,
                     time=from_ts,
                     asset=assets[idx],
-                    amount=str(new_amount),
-                    usd_value=str(action(FVal(assets_data[idx].usd_value), value)),
+                    amount=FVal(new_amount),
```

Comment:
you don't need this ... you are already dealing with FVals
```suggestion
                    amount=new_amount,
```